apps/system/utils/utils_pmsleep.c

```diff
@@ -0,0 +1,30 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <arch/chip/pm.h>
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+int utils_pmsleep(int argc, char **args)
+{
+	return pm_sleep();
```

Comment:
When we call the pm_sleep, does it work always regardless of device status?

Reply:
removed in later commits